src/utils/columnUtils.js

```diff
@@ -25,7 +25,14 @@ export function getColumnProperties(rowProperties, allColumns=[]) {
     // build one object that contains all of the column properties keyed by id
     children.reduce((previous, current, i) => {
       if (current) {
-        previous[current.props.id] = {order: offset + i, ...current.props};
+        if(Array.isArray(current)){
+          current.forEach((c)=>{
+            previous[c.props.id] = {order: offset + i, ...c.props};
```

Comment:
This will end up producing multiple columns that share an `order` value, which can produce unexpected behavior.

Reply:
change to _.flatten also solve this problem, thx.